### DISCUSSION TITLE
Speaker Diarization - based on voice features not order.

### DISCUSSION THREAD

**Discussion by mosesab** [deepgram/community#901] [General help] [answered]
The live transcription does not fit my below usecase because it requires a consistent connection: 
I have used deepgram python-sdk "prerecorded" to load chunks of a large audiofile, i want the speaker diarize to remain consistent between calls. 
How can i do this?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment, marked as answer]
Hi @mosesab 

The short answer is you can't with prerecorded. That wasn't it's intended functionality and use case. You need to use websocket/live streaming capabilities OR wait until the entire conversation recording is available and then use prerecorded.

**Reply by mosesab** [reply]
Thank you for responding, it seems i'll just have to wait for the entire recording
